### ISSUE TITLE
SDWebImage.h was renamed to SDWebImage-umbrella.h without update in docs

### ISSUE THREAD

**Issue by Behinder** [SDWebImage/SDWebImage#2850]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by dreampiggy** [comment]
This umbrella header is generated by CocoaPods. And we have `SDWebImage.h` always here, no removing.

You should list your environment, at least:

1. SDWebImage version ?
2. Installation type: CocoaPods (whether use_frameworks, use_modular_headers, or nother), Carthage, SwiftPM ?

**Comment by dreampiggy** [comment]
Maybe you means this change affect you ? But what the result ?
#2749

**Comment by Behinder** [comment]
Latest version. Cocoapods.
Use modular commented.
Never mind. Docs are difficult to read and not cover my cases so I won’t
use that library anyway.

**Comment by dreampiggy** [comment]
@Behinder I can not reproduce this issue. Using a simple new created demo.

![image](https://user-images.githubusercontent.com/6919743/65482135-9b31c100-deca-11e9-8887-ea5f225b739b.png)
![image](https://user-images.githubusercontent.com/6919743/65482147-a258cf00-deca-11e9-94cd-836ca29a3248.png)
